chameleonultragui/macos/Runner.xcodeproj/xcshareddata/xcschemes/Runner.xcscheme

```diff
@@ -48,6 +48,7 @@
       ignoresPersistentStateOnLaunch = "NO"
       debugDocumentVersioning = "YES"
       debugServiceExtension = "internal"
+      enableGPUValidationMode = "1"
```

Comment:
Why change/add this? did i miss smthng?

Reply:
It's updated by Flutter CLI, seems related to debugging